pkg/provider/azure_loadbalancer.go

```diff
@@ -2815,6 +2816,39 @@ func (az *Cloud) getExpectedLBRules(
 	isIPv6 bool,
 ) ([]*armnetwork.Probe, []*armnetwork.LoadBalancingRule, error) {
 	var expectedRules []*armnetwork.LoadBalancingRule
+	// If we are using Pod IP in the LB backend, we skip health probes, disable floating IP and use port.TargetPort.
+	if az.IsLBBackendPoolTypePodIP() {
+		// Check for multi-IP families in the service spec (not allowed for BackendPool of type PodIP).
+		if len(service.Spec.IPFamilies) > 1 {
+			return nil, nil, fmt.Errorf("dual-stack services are not supported when LB backend pool type is PodIP")
```

Comment:
In the design doc it is said "For dual stack, 2 services need to be created". Can you double check which one is the right behavior?